tests/e2e/pact/broker/verify.ts

```diff
@@ -123,7 +129,53 @@ async function verifyTracker(): Promise<void> {
   await verifyOnce("tracker")
 }
 
+function clearData() {
+  const releaseUrl = `${process.env.PACT_PROVIDER_URL}${basePath}/Task/$release`
+
+  const nominatedReleases = fetcher.taskExamples
+    .filter(task => task.isSuccess)
+    .filter(task => task.requestFile.operation === "release")
+    .map(task => task.request as fhir.Parameters)
+    .filter(isNominatedRelease)
+
+  nominatedReleases.forEach(async release => {
+    let response
+    do {
+      console.log(
+        "Clearing Prescriptions For: ",
+        getIdentifierParameterByName(release.parameter, "owner").valueIdentifier.value
+      )
+      response = await nominatedRelease(releaseUrl, release)
+    }
+    while ((response && response.data.resourceType !== "OperationOutcome") && !isSandbox)
```

Comment:
Feels a bit weird to have the sandbox condition here. Could we not just exit right at the start of the method if we're in sandbox?